views/blog.ejs

```diff
@@ -34,21 +46,53 @@
             <div class="container">
                 <a href="/blogs/explore" class="brand-logo left-align"
                     style="font-family: 'Dancing Script', cursive">Write.it</a>
-                <ul class="right hide-on-med-and-down">
+                <!-- <ul class="right hide-on-med-and-down">
```

Comment:
If you're not planning on using this code, then you should delete it rather than commenting it.